packages/backend/src/services/GitIntegrationService/GitIntegrationService.ts

```diff
@@ -454,14 +447,25 @@ Affected charts:
         return githubProps;
     }
 
-    async createPullRequestForCustomMetrics(
+    async createPullRequest(
         user: SessionUser,
         projectUuid: string,
-        customMetrics: AdditionalMetric[],
         quoteChar: `"` | `'`,
+        args:
```

Comment:
It would be great if we could have supported both side by side. Not a blocker for now.

Reply:
@ZeRego 
> It would be great if we could have supported both side by side. Not a blocker for now.

for the modal where you select multiple right? I was thinking the same